src/functions-and-arrays.js

```diff
@@ -1,20 +1,40 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  return Math.max(num1, num2);
+}
+maxOfTwoNumbers(43, 24)
```

Comment:
After you finish the exercise, project etc - try to delete any unused lines of code. We only had to define the functions, no need to call them ;)